spec/producer_spec.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe FastlyNsq::Producer do
+  let!(:topic) { 'fnsq' }
+  subject { FastlyNsq::Producer.new(topic: topic) }
+
+  before { reset_topic(topic) }
```

Comment:
force reset here and in the `let!`?

Reply:
I'm not seeing the reset inside a let!, can you link it?

Reply:
Perhaps I misinterpreted this block of text from the rspec tests.
```
Note that let is lazy-evaluated: it is not evaluated until the first time
the method it defines is invoked. You can use let! to force the method's
invocation before each example.
```

nm